salt/returners/rawfile_json.py

```diff
@@ -62,7 +59,7 @@ def returner(ret):
     try:
         with salt.utils.files.flopen(opts["filename"], "a") as logfile:
             salt.utils.json.dump(ret, logfile)
-            logfile.write(str("\n"))  # future lint: disable=blacklisted-function
+            logfile.write("\n")  # future lint: disable=blacklisted-function
```

Comment:
```suggestion
            logfile.write("\n")
```
I have no clue why we have these `# future lint: disable=blacklisted-function` on several modules, I think they are related to py2 compat stuff, let's nuke all occurrences of it throughout all of salt's codebase?